README.md

```diff
@@ -25,6 +27,86 @@ This installation process is opinionated in order to get you going quickly. We w
 1. Navigate to `/freestyle`. You should now see something like:
     ![](public/freestyle-generated.png)
 
+## Introduction
+
+### Anatomy of a Basic Style Guide
+
+Here is a simple style guide, where `{{loading-spinner}}` is a hypothetical component **in your application**.
+
+```hbs
+{{#freestyle-guide title="My Living Style Guide" subtitle="Showcasing My App's Components"}}
+  {{#freestyle-section name="UI Elements"}}
+    {{#freestyle-usage "loading-spinner" title="Loading Spinner"}}
+      {{loading-spinner}}
+    {{/freestyle-usage}}
+  {{/freestyle-section}}
+{{/freestyle-guide}}
+```
+
+### Components
+
+Here's a brief rundown of the components Ember Freestyle provides for adding a living style guide in your app:
+
+#### freestyle-guide
+
+The `freestyle-guide` component provides the user interface for a style guide. It includes a header section and
+navigation controls.
+
+#### freestyle-usage
+
+The `freestyle-usage` component is the workhorse. Wrap your application's components with a `freestyle-usage`
+component, being sure to provide a unique slug (positional param) as follows:
+
+```hbs
+{{#freestyle-usage "globally-unique-slug" title="Title To Display In Style Guide"}}
+  {{x-foo propa="aaa" propb="bbb"}}
+{{/freestyle-usage}}
+```
+
+The snippet above will render your app's `x-foo` component as well as a handlebars snippet demonstrating how to use it.
+
+#### freestyle-section
+
+Optionally group your `freestyle-usage`-wrapped components into sections using the `freestyle-section` component. The
+`freestyle-section` component registers itself in order to appear in the navigation provided by the `freestyle-guide`
+component.
+
+#### freestyle-subsection
+
+Optionally divide your style guide sections into subsections using the `freestyle-subsection` component.
+
+```hbs
+{{#freestyle-guide title="My Living Style Guide" subtitle="Showcasing My App's Components"}}
+  {{#freestyle-section name='Visual Style' as |section|}}
+    {{#freestyle-subsection name='Typography' section=section}}
```

Comment:
No, just out of convenience when spiking. I'm hoping to remove this requirement using the `hash` helper.